app/representers/api/auth/account_representer.rb

```diff
@@ -1,14 +1,13 @@
 # encoding: utf-8
 
 class Api::Auth::AccountRepresenter < Api::AccountRepresenter
-
   # point to authorized stories (including unpublished)
   link :stories do
     {
       href: "#{api_authorization_account_stories_path(represented)}{?page,per,zoom,filters}",
       templated: true,
       count: represented.all_stories.count
-    }
+    } if represented.id
```

Comment:
What is this doing?  Only displaying the link at `/accounts/:id`, vs on `/accounts`?